settings/dconf-defaults/locks/00_settings

```diff
@@ -0,0 +1,9 @@
+# This file specifies gsettings keys which are immutable.
+# https://help.gnome.org/admin/system-admin-guide/stable/dconf-lockdown.html.en
+
+# Endless OS users should not be able to enable or disable system extensions.
+# This also have the side-effect that users won't be able to run their own extensions,
+# but we are still going on with that decision.
+# https://phabricator.endlessm.com/T28777
+/org/gnome/shell/enabled-extensions
```

Comment:
You still need to lock `disabled-extensions` and `disable-user-extensions`. It's not in the System Admin Guide, but it's still necessary.

Reply:
Addressed.